app/src/main/res/values/strings.xml

```diff
@@ -214,7 +214,7 @@
   <string name="patientsummary_toolbar_title">%1$s, %2$s, %3$s</string>
   <string name="patientsummary_bangladesh_national_id">ID</string>
 
-  <string name="patientsummary_contact_doctor">Doctor</string>
+  <string name="patientsummary_contact_doctor" translatable="false">Tele-consult</string>
```

Comment:
There is no point settings this string as `translatable="false"`, we can create a new string called `patientsummary_teleconsult` and remove this string in `Remove unused resources` commit.